can not select existing controller to deploy to

```
2017-06-19 14:09:54,223 [DEBUG] conjure-up/openstack-novalxd - common.py:67 - LXD Init: <bound method BaseLXDSetupController.set_lxd_init_auto of <conjureup.controllers.lxdsetup.gui.LXDSetupController object at 0x7fba5e5ed828>>
2017-06-19 14:09:54,271 [DEBUG] conjure-up/openstack-novalxd - common.py:67 - LXD Init: <bound method BaseLXDSetupController.set_lxc_config of <conjureup.controllers.lxdsetup.gui.LXDSetupController object at 0x7fba5e5ed828>>
2017-06-19 14:09:54,295 [DEBUG] conjure-up/openstack-novalxd - common.py:67 - LXD Init: <bound method BaseLXDSetupController.set_lxd_storage of <conjureup.controllers.lxdsetup.gui.LXDSetupController object at 0x7fba5e5ed828>>
2017-06-19 14:09:54,316 [DEBUG] conjure-up/openstack-novalxd - common.py:67 - LXD Init: functools.partial(<bound method BaseLXDSetupController.setup_bridge_network of <conjureup.controllers.lxdsetup.gui.LXDSetupController object at 0x7fba5e5ed828>>, 'ens3')
2017-06-19 14:09:54,342 [DEBUG] conjure-up/openstack-novalxd - common.py:67 - LXD Init: <bound method BaseLXDSetupController.setup_unused_bridge_network of <conjureup.controllers.lxdsetup.gui.LXDSetupController object at 0x7fba5e5ed828>>
2017-06-19 14:09:54,367 [DEBUG] conjure-up/openstack-novalxd - common.py:67 - LXD Init: <bound method BaseLXDSetupController.set_default_profile of <conjureup.controllers.lxdsetup.gui.LXDSetupController object at 0x7fba5e5ed828>>
2017-06-19 14:09:56,447 [INFO] conjure-up/openstack-novalxd - common.py:77 - Bootstrapping Juju controller.
2017-06-19 14:09:56,448 [DEBUG] conjure-up/openstack-novalxd - juju.py:170 - bootstrap cmd: ['juju', 'bootstrap', 'localhost', 'conjure-up-controller', '--default-model', 'conjure-openstack-novalx-3eb', '--config', 'image-stream=daily', '--config', 'enable-os-upgrade=false', '--bootstrap-series', 'xenial']
2017-06-19 14:09:56,463 [DEBUG] conjure-up/openstack-novalxd - events.py:50 - Awaiting ModelConnected at conjureup/controllers/deploy/common.py:23
2017-06-19 14:09:56,494 [DEBUG] conjure-up/openstack-novalxd - juju.py:179 - waiting for proc
2017-06-19 14:09:57,088 [DEBUG] conjure-up/openstack-novalxd - juju.py:181 - proc done
2017-06-19 14:09:57,088 [ERROR] conjure-up/openstack-novalxd - common.py:36 - Error bootstrapping controller: ['ERROR controller "conjure-up-controller" already exists']
2017-06-19 14:09:57,089 [DEBUG] conjure-up/openstack-novalxd - events.py:50 - Setting Error at conjureup/events.py:133
2017-06-19 14:09:57,155 [ERROR] conjure-up/openstack-novalxd - events.py:152 - Unhandled exception
Traceback (most recent call last):
  File "/snap/conjure-up/452/lib/python3.6/site-packages/conjureup/controllers/bootstrap/common.py", line 39, in do_bootstrap
    app.current_cloud_type))
Exception: Unable to bootstrap (cloud type: localhost)

```

Selecting an existing controller will fail as we attempt to bootstrap a new controller rather than doing an add model

--- a/conjureup/controllers/controllerpicker/common.py
+++ b/conjureup/controllers/controllerpicker/common.py
@@ -30,7 +30,7 @@ class BaseControllerPicker:
         else:
             app.current_controller = controller
 
-        if app.current_controller not in juju.get_controllers():
+        if app.current_controller not in juju.get_controllers()['controllers']:
             return controllers.use('bootstrap').render()
 
         c_info = juju.get_controller_info(app.current_controller)

